Packs/IdentityRecordedFuture/Integrations/IdentityRecordedFuture/IdentityRecordedFuture.py

```diff
@@ -211,28 +217,33 @@ def playbook_alert_update_command(self) -> Optional[List[CommandResults]]:
         response = self.client.update_playbook_alerts()
         return self._process_result_actions(response=response)
 
+    @staticmethod
+    def _transform_incidents_attachments(incidents: list) -> None:
+        for incident in incidents:
+            attachments = []
+            incident_json = json.loads(incident.get("rawJSON", "{}"))
+            if incident_json.get("panel_evidence_summary", {}).get("screenshots"):
+                for screenshot_data in incident_json["panel_evidence_summary"][
```

Comment:
The code will throw here an exception if incident_json["panel_evidence_summary"]["screenshots"] does not exist. Is that ok for you?

Reply:
I reviewed all the places and I'm okay with them, they are pretty good covered in case of missing key